src/main/kotlin/no/nav/familie/ba/sak/kjerne/brev/BrevService.kt

```diff
@@ -343,6 +348,70 @@ class BrevService(
+            sjekkOmDetErLøpendeDifferanseUtbetalingPåBehandling(behandling)
+        }
+
+        val vedtakFellesfelterSammensattKontrollsak = lagVedtaksbrevFellesfelterSammensattKontrollsak(vedtak = vedtak, sammensattKontrollsak = sammensattKontrollsak)
+        return when (brevmal) {
+            Brevmal.VEDTAK_OPPHØRT -> {
+                OpphørtSammensattKontrollsak(
+                    vedtakFellesfelterSammensattKontrollsak = vedtakFellesfelterSammensattKontrollsak,
+                    erFeilutbetalingPåBehandling = erFeilutbetalingPåBehandling(behandlingId = behandling.id),
+                )
+            }
+
+            Brevmal.VEDTAK_OPPHØR_MED_ENDRING -> {
+                OpphørMedEndringSammensattKontrollsak(
+                    vedtakFellesfelter = vedtakFellesfelterSammensattKontrollsak,
+                    etterbetaling = hentEtterbetaling(vedtak),
+                    erFeilutbetalingPåBehandling = erFeilutbetalingPåBehandling(behandlingId = behandling.id),
+                    refusjonEosAvklart = beskrivPerioderMedAvklartRefusjonEøs(vedtak),
+                    refusjonEosUavklart = beskrivPerioderMedUavklartRefusjonEøs(vedtak),
+                    erKlage = behandling.erKlage(),
+                    utbetalingstabellAutomatiskValutajustering = hentLandOgStartdatoForUtbetalingstabell(vedtak, vedtakFellesfelterSammensattKontrollsak.utbetalingerPerMndEøs),
+                )
+            }
+
+            Brevmal.VEDTAK_ENDRING -> {
+                VedtakEndringSammensattKontrollsak(
+                    vedtakFellesfelter = vedtakFellesfelterSammensattKontrollsak,
+                    etterbetaling = hentEtterbetaling(vedtak),
+                    erKlage = behandling.erKlage(),
+                    erFeilutbetalingPåBehandling = erFeilutbetalingPåBehandling(behandlingId = behandling.id),
+                    informasjonOmAarligKontroll = vedtaksperiodeService.skalHaÅrligKontroll(vedtak),
+                    feilutbetaltValuta =
+                        vedtaksperiodeService.beskrivPerioderMedFeilutbetaltValuta(vedtak)?.let {
+                            FeilutbetaltValuta(perioderMedForMyeUtbetalt = it)
+                        },
+                    refusjonEosAvklart = beskrivPerioderMedAvklartRefusjonEøs(vedtak),
+                    refusjonEosUavklart = beskrivPerioderMedUavklartRefusjonEøs(vedtak),
+                    duMåMeldeFraOmEndringer = !skalMeldeFraOmEndringerEøsSelvstendigRett,
+                    duMåMeldeFraOmEndringerEøsSelvstendigRett = skalMeldeFraOmEndringerEøsSelvstendigRett,
+                    informasjonOmUtbetaling = skalInkludereInformasjonOmUtbetaling,
+                    utbetalingstabellAutomatiskValutajustering = hentLandOgStartdatoForUtbetalingstabell(vedtak, vedtakFellesfelterSammensattKontrollsak.utbetalingerPerMndEøs),
+                )
+            }
+
+            else -> throw Feil("Brevmalen: $brevmal er ikke støttet for sammensatte kontrollsaker")
```

Comment:
gir kolonet mening her?

```suggestion
            else -> throw Feil("Brevmalen $brevmal er ikke støttet for sammensatte kontrollsaker")
```